src/modules/fundingManager/oracle/interfaces/ILM_ManualExternalPriceSetter_v1.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Imports
+
+// Internal
+import {IOraclePrice_v1} from
+    "src/modules/fundingManager/oracle/interfaces/IOraclePrice_v1.sol";
+
+/**
+ * @title   Manual External Price Oracle Interface
+ *
+ * @notice  Interface for the manual price feed mechanism that allows setting
+ *          and updating prices for token issuance and redemption operations.
+ *
+ * @dev     This interface extends IOraclePrice_v1 and adds functionality for
+ *          manually setting prices. Both prices (issuance and redemption) must
+ *          be non-zero values and can only be set by the contract owner.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to
+ *                          our Security Policy at security.inverter.network or
+ *                          email us directly!
+ *
+ * @author  Zealynx Security
+ */
+interface ILM_ManualExternalPriceSetter_v1 is IOraclePrice_v1 {
+    //--------------------------------------------------------------------------
+    // Events
+
+    /// @notice Emitted when an issuance price is set
+    /// @param  price The new price that was set
+    event IssuancePriceSet(uint indexed price);
```

Comment:
We also need to emit the timestamp of when the price has been updated for both issuance and redemption